drivers/adc/adc_smartbond_sdadc.c

```diff
@@ -119,9 +119,7 @@ static inline void sdadc_smartbond_pm_policy_state_lock_get(const struct device
 					      struct sdadc_smartbond_data *data)
 {
 #if defined(CONFIG_PM_DEVICE)
-#if defined(CONFIG_PM_DEVICE_RUNTIME)
 	pm_device_runtime_get(dev);
-#endif
 
 	if (!atomic_test_and_set_bit(data->pm_policy_state_flag, 0)) {
```

Comment:
See relative [comment ](https://github.com/zephyrproject-rtos/zephyr/pull/74009#discussion_r1643499751)for the GPADC driver.

Reply:
Removed atomic operation.